products/binaryauthorization/terraform.yaml

```diff
@@ -38,6 +39,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+        custom_expand: 'templates/terraform/custom_expand/self_link_from_name.erb'
```

Comment:
I'm very surprised that the tests pass with this. It has references to `/compute/v1/` URLs, which seems like it would break this. It also looks like it would fall back to adding `"{{ComputeBasePath}}projects/{{project}}/regions/{{region}}/")` onto the the front of the name, which doesn't make sense for attestor which doesn't have a region.

Is it possible to just write a custom expander similar to `repository_name_from_short_name`?

Reply:
Good catch!  Sorry about that!  I need to slow down and double-check things :)